### PR TITLE
add hardware encoding examples

### PR DESCRIPTION
**Pull request by axfelix**
## Checklist

* [x] I've referred to the [Guidelines for contributing](https://github.com/amiaopensource/ffmprovisr/blob/gh-pages/readme.md#guidelines-for-contributing)

Hi, here's a GPU encoding example, it's not too well-documented elsewhere and I use this from time to time so it seemed worth getting in here!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -337,6 +337,37 @@ <h5>Transcode to H.264</h5>
...
+    <div class="hiding">
+      <h5>Transcode to H.264/H.265 using the GPU</h5>
+      <p><code>ffmpeg -i <em>input_file</em> -c:v h264_nvenc -preset llhq -rc:v vbr_hq -cq:v 19 -b:v 8000k -maxrate:v 12000k -profile:v high -c:a copy <em>output_file</em></code></p>
+      <p>This command takes an input file and transcodes it to H.264 using the encoding functionality of an Nvidia GPU (without transcoding the audio). If you're using H.264 with AAC or AC3 audio, you can output to an .mp4 file; if you're using HEVC and/or more exotic audio, you should output to .mkv. While Nvidia's fixed-function hardware can be 10x as performant as encoding on the CPU, it requires a few more parameters in order to optimize quality at lower bitrates.</p>
+      <dl>
+        <dt>ffmpeg</dt><dd>starts the command</dd>
+        <dt>-i <em>input_file</em></dt><dd>path, name and extension of the input file</dd>
+        <dt>-c:v <em>h264_nvenc</em></dt><dd>tells FFmpeg to encode the video stream as H.264 using Nvidia's encoder.</dd>
+        <dt>--preset <em>llhq</em></dt><dd>uses the "low latency, high quality" encoding preset, a good default when working with nvenc.</dd>
```
Comment (retokromer) [inline index.html]: `--preset` > `-preset`

Comment (axfelix) [reply index.html]: thx, good catch

File: recipes.txt [hunk 2]
```diff
@@ -16,6 +16,8 @@ ffmpeg -i input_file -map 0 -dn -c:v ffv1 -level 3 -g 1 -slicecrc 1 -slices 16 -
 ffmpeg -i concat:input_file_1\|input_file_2\|input_file_3 -c:v libx264 -c:a aac output_file.mp4
 # Transcode to an H.265/HEVC MP4
 ffmpeg -i input_file -c:v libx265 -pix_fmt yuv420p -c:a copy output_file
+# Transcode to H.264/H.265 using the GPU
+ffmpeg -i input_file -c:v h264_nvenc -preset llhq -rc:v vbr_hq -cq:v 19 -b:v 8000k -maxrate:v 12000k -profile:v high -c:a copy output_file
```
Comment (retokromer) [inline recipes.txt:20]: You could add both recipes the one for H.264 and the one for H.265. That way both can be imported easily into FFCommand Engine.

**Review by retokromer — APPROVED** [review]
One nit, otherwise reading the code LGTM. I can test tomorrow, if nobody does it quicker.

**Comment by retokromer** [comment]
Oh, already fixed!

**Review by ablwr — APPROVED** [review]

**Comment by ablwr** [comment]
Niiiicccceeeee work

**Comment by axfelix** [comment]
Nit picked, thx for the suggestion, actually caught a bug in the HEVC command... I don't encode to HEVC very often still.

**Comment by retokromer** [comment]
Thank you, @axfelix !

**Pull request merged**